SocialCareCaseViewerApi/V1/Boundary/Requests/CreateCaseStatusRequest.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using FluentValidation;
+using SocialCareCaseViewerApi.V1.Domain;
+
+namespace SocialCareCaseViewerApi.V1.Boundary.Requests
+
+#nullable enable
+{
+    public class CreateCaseStatusRequest
+    {
+        [JsonPropertyName("personId")]
+        public long PersonId { get; set; }
+
+        [JsonPropertyName("type")]
+        public string Type { get; set; } = null!;
+        [JsonIgnore]
+        public long? TypeId { get; set; }
+
+        [JsonPropertyName("fields")]
+        public List<CaseStatusRequestField>? Fields { get; set; } = new List<CaseStatusRequestField>() { }!;
```

Comment:
`public List<CaseStatusRequestField> Fields { get; set; } = new List<CaseStatusRequestField>();`

this might be more sensible, by default this says if nothing is passed in the fields property it is set to an empty array. this also means that it isn't nullable, otherwise we need some null checks in other places in the code or risk some null reference exceptions which result in a 500 response